packages/grafana-runtime/rollup.config.ts

```diff
@@ -20,7 +20,7 @@ const buildCjsPackage = ({ env }) => {
         globals: {},
       },
     ],
-    external: ['lodash', '@grafana/ui', '@grafana/data'], // Use Lodash from grafana
+    external: ['lodash', '@grafana/ui', '@grafana/data', '@grafana/e2e-selectors'], // Use Lodash from grafana
```

Comment:
This is correct, right @dprokop @jackw 🙏

Reply:
Should be correct otherwise rollup will try to bundle e2e-selectors in the runtime package. 👍